packages/l/logseq/package.yml

```diff
@@ -0,0 +1,49 @@
+name       : logseq
+version    : 0.9.20
+release    : 1
+source     :
+    - https://github.com/logseq/logseq/archive/refs/tags/0.9.20.tar.gz : 306a8e7adc78bf62f6c931bdd2a118d39bb437d7dd4cfe9f9193b7548de32e32
+homepage   : https://logseq.com/
+license    : AGPL-3.0
+component  : office
+summary    : A privacy-first, open-source platform for knowledge management and collaboration
+description: |
+    Logseq is a knowledge management and collaboration platform. It focuses on privacy, longevity, and user control. Logseq offers a range of powerful tools for knowledge management, collaboration, PDF annotation, and task management with support for multiple file formats, including Markdown and Org-mode, and various features for organizing and structuring your notes.    
+networking : yes
+builddeps   :
+    - clojure-tools
+    - git
+    - nodejs
+    - yarn
+environment: |
+    export JAVA_HOME=/usr/lib64/openjdk-17
+    export PATH=$JAVA_HOME/bin:$PATH
+setup      : |
+    # patch :parallel-build true in shadow-cljs.edn
+    %patch -p1 -i $pkgfiles/build.patch
+     # download required js modules
+    yarn install
```

Comment:
Use `yarn install --frozen-lockfile` if you can, to avoid different builds ending up with different versions of dependencies.